README.md

```diff
@@ -164,16 +184,35 @@ At the moment, there are four queries:
 
 ```
 {
-   repositoriesByExtensions (extensions: ["quarkus-json", "quarkus-rest"]) {
+  repositoriesByExtensions(extensions: [{name: "quarkus-grpc", version: "1.10.2.Final"}]) {
     id
     repoUrl
     name
+    quarkusVersion
     branch
     labels
     createdAt
     updatedAt
     status
-   }
+  }
+}
+```
+
+- repositories that use any of a list of artifactIds:
+
+```
+{
+  repositoriesByExtensionsArtifactIds(artifactIds: ["quarkus-grpc", "quarkus-jdbc-h2"]) {
```

Comment:
What is the difference between repositoriesByExtensionsArtifactIds and repositoriesByExtensions ? As far I see, the only difference is that this query accepts a list of string and the latter a list of extensions which is more usable and powerful.

Reply:
repositoriesByExtensions -> returns all the repos that contain any of the given extensions (artifactsID + Version).

repositoriesByExtensionsArtifactIds -> returns all the repos that contain any of the artifacts (you don't care about the version). So if you don't really know the version that you are looking for, this is a better query.

Reply:
My point is that why not using directly the first query. If the version is not provided, it will behave as the second query. Still don't see the point of having two queries.